### PR TITLE
Optimize graphics

### PR DESCRIPTION
**Pull request by Shosoul**
This changes optimizes graphics to be able to run calculations faster, especially in loops. Also stops rendering things that we don't use anymore.

### CODE REVIEW

File: graphics.lua [hunk 1]
```diff
@@ -568,57 +570,57 @@ function Stack.render(self)
...
-      if self.maxPrestop > 0 then
-        prestop_bar = self.pre_stop_time * (themes[config.theme].images.IMG_prestop_bar:getHeight() / self.maxPrestop)
-      else
-        prestop_bar = 0
-      end
-      prestopQuad:setViewport(0, themes[config.theme].images.IMG_prestop_bar:getHeight() - prestop_bar, themes[config.theme].images.IMG_prestop_bar:getWidth(), prestop_bar)
-      qdraw(themes[config.theme].images.IMG_prestop_bar, prestopQuad, self.origin_x + (themes[config.theme].prestop_bar_Pos[1] * self.mirror_x), ((self.pos_y + themes[config.theme].prestop_bar_Pos[2]) + ((themes[config.theme].images.IMG_prestop_bar:getHeight() - prestop_bar) / GFX_SCALE)), themes[config.theme].prestop_bar_Rotate, themes[config.theme].prestop_bar_Scale / GFX_SCALE, themes[config.theme].prestop_bar_Scale / GFX_SCALE, 0, 0, self.multiplication)
+      -- prestop_frame_Pos = {(self.origin_x + themes[config.theme].prestop_frame_Pos[1] * self.mirror_x) + ((themes[config.theme].images.IMG_prestop_frame:getWidth() - 10) / GFX_SCALE * self.multiplication * self.mirror_x), self.pos_y + themes[config.theme].prestop_frame_Pos[2]}
```
[the remark this answers is not in the document]
Comment (Shosoul) [reply graphics.lua:573]: I don't think there are any serious themes that use these assets still, I think you may have saw it when the config file was still broken

**Comment by Shosoul** [comment]
Any update for this pull?

**Comment by JamesVanBoxtel** [comment]
I'm not sure we should do this for now. I'm not convinced we should remove those features from themes that might still want to render them, and all the rename changes are going to make potential merge conflicts in telegraph

**Comment by Shosoul** [comment]
If you find a single theme on the mod page that uses this I will close this pull, I honestly don’t think any themes use it anymore.

**Comment by Shosoul** [comment]
Well, telegraph is a different issue, so I’ll close it for that. I still think that no themes use it though

**Pull request closed without merging**